bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/MyStromPlugHandler.java

```diff
@@ -96,6 +98,7 @@ protected void pollDevice() {
         if (report != null) {
             updateState(CHANNEL_SWITCH, report.relay ? OnOffType.ON : OnOffType.OFF);
             updateState(CHANNEL_POWER, QuantityType.valueOf(report.power, WATT));
+            updateState(CHANNEL_WS, QuantityType.valueOf(report.Ws, WATT));
```

Comment:
Please see my other comment. The unit here is most likely wrong - probably it should be `WATT_SECOND`.